README.adoc

```diff
@@ -273,14 +279,16 @@ Tests run: 2, Failures: 0, Errors: 0, Skipped: 0
 
 To see whether the tests detect a failure, change the [hotspot=endpoint file=0]`endpoint` for the `inventory` service in
 the [hotspot file=0]`src/main/java/io/openliberty/guides/inventory/InventoryResource.java` file to something else, then
-run the Maven build again. You see a test failure occur.
+run the tests again to see a test failure occur.
 
 InventoryResource.java
 [source, Java, linenums, role='code_column hide_tags=copyright']
 ----
 include::finish/src/main/java/io/openliberty/guides/inventory/InventoryResource.java[]
 ----
 
+When you are done checking out the service, exit development mode by typing `q` in the shell session where you ran the server and then pressing the `enter/return` key.
+
 == Great work! You're done!
 
 You have just completed building a simple inventory application using CDI services in Open Liberty.
```

Comment:
Change
"You have just completed building a simple inventory application using CDI services in Open Liberty."
To
"You just used CDI services in Open Liberty to build a simple inventory application."